docker/run.sh

```diff
@@ -27,16 +36,32 @@ else
 fi
 
 
-if [[ "$image" == "flexflow-environment" ]]; then
-    eval docker run -it "$gpu_arg" "--shm-size=${SHM_SIZE}" "flexflow-environment-${FF_GPU_BACKEND}:latest"
-elif [[ "$image" == "flexflow" ]]; then
-    eval docker run -it "$gpu_arg" "--shm-size=${SHM_SIZE}" "flexflow-${FF_GPU_BACKEND}:latest"
-elif [[ "$image" == "mt5" ]]; then
-    # Backward compatibility
-    eval docker run -it "$gpu_arg" "--shm-size=${SHM_SIZE}" \
-    -v "$(pwd)"/../examples/python/pytorch/mt5/data:/usr/FlexFlow/examples/python/pytorch/mt5/data \
-    -v "$(pwd)"/../examples/python/pytorch/mt5/eng-sin.tar:/usr/FlexFlow/examples/python/pytorch/mt5/eng-sin.tar \
-    "flexflow-${FF_GPU_BACKEND}:latest"
+if [[ "${FF_GPU_BACKEND}" == "hip_rocm" ]]; then
```

Comment:
I think here, to reduce the code duplication, we could use the `if [[ "${FF_GPU_BACKEND}" == "hip_rocm" ]]; then ... else ... fi` statement only to modify the `cuda_version` variable. For rocm, we can make it an empty string, whereas for CUDA, we can set it as `cuda_version="-${cuda_version}"` (we add the hyphen). Then, we can run the docker machines "flexflow-environment-${FF_GPU_BACKEND}${cuda_version}:latest", "flexflow-${FF_GPU_BACKEND}${cuda_version}:latest", etc... in all cases. Note how I have removed the hyphen between `${FF_GPU_BACKEND}` and `${cuda_version}`